DEVELOPER.md

```diff
@@ -47,12 +47,31 @@ $ cd fluvio
 $ cargo build
 ```
 
-Setup alias for development CLI.
+The next step is very important, as it will help you to prevent subtle development
+bugs. Fluvio is built in two separate pieces, `fluvio` (the CLI), and `fluvio-run`
+(the server). When testing changes to these components, you need to make sure to
+rebuild _both_ components before running. In other Rust projects, it is typical to
+just use `cargo run`:
+
+```
+$ cargo run -- my CLI args here
 ```
-$ alias flvd=./target/debug/fluvio
-``` 
 
-This avoid collision with released version of Fluvio.
+However, this will only rebuild `fluvio`, it will not also rebuild `fluvio-run`,
+which may make you think that the code changes you made did not have any effect.
+In order to automate the rebuilding of both of these components, we STRONGLY
+RECOMMEND adding the following function to your `~/.bashrc` or `~/.zshrc` file:
+
+```
+flvd () {
```

Comment:
let's put this under `cargo run` approach

Reply:
What do you mean?